jamjar/jamjar/videos/views.py

```diff
@@ -3,11 +3,32 @@
 from rest_framework.authtoken.models import Token
 from jamjar.base.views import BaseView
 
+from rest_framework.parsers import FormParser, MultiPartParser
+
 from jamjar.videos.models import Video
 from jamjar.videos.serializers import VideoSerializer
 
-class VideoList(BaseView):
+import uuid
+
+from django.http import HttpResponse
+from wsgiref.util import FileWrapper
+
+
+class VideoStream(BaseView):
+    def get(self, request, id):
+         # Attempt to get the video
+        try:
+            self.video = Video.objects.get(id=id)
+        except:
+            return self.error_response('Video does not exist or you do not have access to this video.', 404)
 
+        file = FileWrapper(open(self.video.src, 'rb'))
```

Comment:
Will the `src` field be the entire path on the server?  Otherwise we should throw a `VIDEOS_PATH` constant into the settings.py and use the `src` as a relative location from that path.